code/modules/mob/living/simple_animal/hostile/mimic.dm

```diff
@@ -47,7 +49,14 @@
 	ghostize()
 	qdel(src)
 
-
+/mob/living/simple_animal/hostile/mimic/emp_act(severity)
+	if(is_electronic)
+		switch(severity)
+			if(1)
+				Die()
+			if(2)
+				adjustBruteLoss(50)
+	..()

```

Comment:
This will _probably_ work without including severity in the arguments, but you can never be sure, so it should really be ..(severity)